provision/testdata/flavor_calico-3.23.2.apic.txt

```diff
@@ -341,11 +341,11 @@
         }
     }
 }
-/api/mo/uni/tn-test/flt-calico-l3out-fsvi-vl-allow-all-filter.json
+/api/mo/uni/tn-test/flt-calico-l3out-fsvi-vlan-13-allow-all-filter.json
 {
     "vzFilter": {
         "attributes": {
-            "name": "calico-l3out-fsvi-vl-allow-all-filter",
+            "name": "calico-l3out-fsvi-vlan-13-allow-all-filter",
```

Comment:
I am wondering where the fsvi is coming from? Also a bit worried that the suffix and prefix in the names seem to be too long (only the "vlan-13" part is dynamic here?).

Reply:
will be taking this up as a separate PR